### PR TITLE
Support adding events to Outlook calendar

### PR DESCRIPTION
**Pull request by GovernmentPlates**
Closes: #6075 

This PR adds the ability to add events directly to Outlook as well as introduces improvements to the social widget:
- Remove Facebook Like and Google+ sharing options
- Replace old social icon with a generic share icon (from icomoon)
- General UI tweaks

Preview:
image
image

### CODE REVIEW

File: indico/modules/events/client/js/util/social.js [hunk 1]
```diff
@@ -70,4 +55,4 @@
 
     container.delay(250).fadeIn(1000);
   });
-})();
+})();
```
Comment (ThiefMaster) [inline indico/modules/events/client/js/util/social.js:58]: you removed the trailing linebreak

File: indico/modules/events/templates/footer.html [hunk 2]
```diff
@@ -48,13 +30,18 @@ <h4>{% trans %}Calendaring{% endtrans %}</h4>
                     <img src="{{ url_for('assets.image', filename='google_calendar.gif') }}" alt="" border="0">
                 </a>
             </div>
+            <div class="social-site">
+                <a href="https://outlook.office.com/calendar/deeplink/compose?{{ outlook_calendar_params|urlencode }}&path=%2Fcalendar%2Faction%2Fcompose&rru=addevent" target="_blank">
```
Comment (ThiefMaster) [inline indico/modules/events/templates/footer.html:34]: What about adding `&path=%2Fcalendar%2Faction%2Fcompose&rru=addevent` to the `outlook_calendar_params` to have everything in one place?

File: indico/modules/events/client/js/util/social.js [hunk 3]
```diff
@@ -55,8 +42,6 @@
       },
       events: {
         render() {
-          injectFacebook($('.social-button-container').data('social-settings').facebook_app_id);
-          $.getScript('//apis.google.com/js/plusone.js');
           $.getScript('//platform.twitter.com/widgets.js');
```
Comment (ThiefMaster) [inline indico/modules/events/client/js/util/social.js:45]: Is this still needed?

File: indico/modules/events/templates/footer.html [hunk 4]
```diff
@@ -16,26 +16,8 @@ <h4>{% trans %}Direct link{% endtrans %}</h4>
...
-                </div>
-                <div class="social-site">
-                    <div class="g-plusone"
-                         data-annotation="inline"
-                         data-width="220"
-                         data-href="{{ url }}"></div>
-                </div>
                 <div class="social-site">
-                    <a href="https://twitter.com/share" class="twitter-share-button" data-count="horizontal">
+                    <a href="https://twitter.com/intent/tweet?text={{ url }}" class="twitter-share-button">
```
Comment (ThiefMaster) [inline indico/modules/events/templates/footer.html:20]: Would be nice to keep the event title in the text as well (like we have right now)

**Comment by ThiefMaster** [comment]
I think this widget deserves a bigger makeover at some point (it still doesn't look super amazing).

Anyway, for the upcoming release your cleanups are nice and the Outlook share will certainly be appreciated -> I'm good to merge it tomorrow once the comments are resolved.

File: indico/modules/events/client/js/util/social.js (hunk 3 above)
Comment (GovernmentPlates) [reply indico/modules/events/client/js/util/social.js]: Now has been replaced with a 'spyware free' button

**Review by ThiefMaster — APPROVED** [review]

**Pull request merged**